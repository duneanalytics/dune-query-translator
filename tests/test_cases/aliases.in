SELECT 1 AS a, 2 AS b, 3 AS c GROUP BY a, b, c ORDER BY a, b, c