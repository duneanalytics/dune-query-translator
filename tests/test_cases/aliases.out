SELECT
  1 AS a,
  2 AS b,
  3 AS c
GROUP BY
  1,
  2,
  3
ORDER BY
  a,
  b,
  c